docs/ADR/0048-class-side-method-syntax.md

```
@@ -0,0 +1,567 @@
+    new => self error: 'Use uniqueInstance instead'
+    uniqueInstance =>
+        self.uniqueInstance ifNil: [self.uniqueInstance := super new].
+        self.uniqueInstance
+]
+```
+
+### User-Facing Communication
+
+Because this is a syntax breaking change, migration must be coordinated with a compiler version increment. The compiler should emit a clear diagnostic for any source file that still uses the old `class methodName` modifier form after the migration deadline:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)